src/components/Autocomplete/SectionItem/SectionItem.tsx

```diff
@@ -11,11 +12,40 @@ export interface SectionItemProps {
   displaySearchTermHighlights?: boolean;
 }
 
+export function getSearchSuggestionFeatures(request: Partial<AutocompleteRequestType>) {
```

Comment:
I think this need to be part of the hooks logic and we only expose the features as a state returned form the hooks instead of exposing the utility function for consumers of the library to use it to get the features. And we do call `getSearchSuggestionFeatures` internally instead and expose smth like this in the hook response
```
featureToggles: {
    featureDisplaySearchSuggestionImages,
    featureDisplaySearchSuggestionResultCounts,
  }
```

@esezen what do you think?

Reply:
I think this makes sense. @stanlp1 anything against this approach?

Reply:
Nope, Islam mentioned this to me, I suggested tagging you as the decision maker 😂